package.json

```diff
@@ -10,73 +10,89 @@
     "build": "ember build",
     "start": "ember server",
     "test": "npm run lint && ember test",
-    "eslint": "eslint *.js addon app blueprints config tests",
+    "eslint": "eslint *.js addon app blueprints config tests --fix",
     "sass-lint": "sass-lint -q -v",
     "lint": "npm run eslint && npm run sass-lint"
   },
   "repository": "git@github.com:ciena-frost/ember-frost-modal.git",
   "engines": {
     "node": ">= 5.0.0"
   },
-  "author": "Roxanne Panthaky (https://github.com/rox163)",
+  "author": "Steven Glanzer (https://github.com/sglanzer)",
   "contributors": [
     "Eric White (https://github.com/EWhite613)",
     "Gosia Hyndman (https://github.com/vesper2000)",
     "Justin Lafleur (https://github.com/laflower)",
     "Lisa Irwin (https://github.com/lirwin)",
-    "Matthew Dahl (https://github.com/sandersky)"
+    "Matthew Dahl (https://github.com/sandersky)",
+    "Roxanne Panthaky (https://github.com/rox163)"
   ],
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "ember-ajax": "0.7.1",
-    "ember-block-slots": "0.12.3",
-    "ember-browserify": "1.1.9",
-    "ember-cli": "~2.3.0",
+    "ember-browserify": "1.1.12",
+    "ember-cli": "2.6.3",
     "ember-cli-app-version": "^1.0.0",
+    "ember-cli-code-coverage": "git://github.com/kategengler/ember-cli-code-coverage.git#ac2be3fd751a5667f3e4ccf0d5e9dc3ec0936da1",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
+    "ember-cli-import-polyfill": "0.2.0",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mirage": "~0.1.11",
     "ember-cli-mocha": "^0.10.0",
-    "ember-cli-notifications": "^3.2.0",
+    "ember-cli-moment-shim": "2.0.0",
+    "ember-cli-notifications": "3.3.1",
     "ember-cli-release": "0.2.8",
-    "ember-cli-sass": "5.3.0",
-    "ember-cli-sri": "^2.0.0",
+    "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-cli-visual-acceptance": "^1.3.17",
+    "ember-code-snippet": "1.3.0",
     "ember-computed-decorators": "0.2.2",
-    "ember-data": "~2.3.0",
+    "ember-concurrency": "0.7.8",
     "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-elsewhere": "~0.3.0",
```

Comment:
I dont think so because we setup the blueprints for this addon to add this and other packages accordingly.